tests/Microsoft.AspNet.SignalR.Redis.Tests/RedisMessageBusFacts.cs

```diff
@@ -65,9 +65,10 @@ public void OpenCalledOnConnectionRestored()
                 }
             });
 
-            // Creating an instance to invoke the constructor which starts the connection
             var instance = redisMessageBus.Object;
+            await instance.ConnectWithRetry();
```

Comment:
Because we now don't raise the ConnectionRestored event when the connection is in the open state (to avoid duplicate events) this test stopped working. It relied on the fact that ConnectionRestored/ConnectionFailed will be fired regardless of the state the connection is in.